client/src/containers/NewSongInputProvider.js

```diff
@@ -1,22 +1,22 @@
 import {connect} from "react-redux/es/alternate-renderers";
 import {bindActionCreators} from "redux/es/redux";
 import {addSong, updateLink,} from "../actions/actions";
-import {NewSongInput} from "../components/NewSongInput";
+import NewSongInput from "../components/NewSongInput";
 
-function mapStateToProps(state) {
+const mapStateToProps = state => {
```

Comment:
you can wrap the object with () and not specify the return:
```js
const example = () => ({
    a: 1,
    b:2
});

example() // { a:1 , b:2 }

Reply:
its more readable this way don't you think?

Reply:
it's the convention in js so no - when i look on this function i know right away it returns an object